queue_job/readme/CONFIGURE.rst

```diff
@@ -41,3 +41,14 @@
 
 .. [1] It works with the threaded Odoo server too, although this way
        of running Odoo is obviously not for production purposes.
+
+* Be sure to check out *Jobs Garbage Collector* CRON and change *enqueued_delta* and *started_delta* parameters to your needs.
+
+  * enqueued_delta: Spent time in minutes after which an enqueued job is considered stuck.
```

Comment:
On a well configured system where the capacity of the root channel is smaller than the number of available workers, this value should be very short (typically a few seconds).

Reply:
You can't specify seconds because the value are minutes, right? But in any case it's a comment worth including in the readme.